components/aws/sagemaker/batch_transform/src/batch_transform.py

```diff
@@ -62,7 +62,14 @@ def main(argv=None):
   logging.info('Submitting Batch Transformation request to SageMaker...')
   batch_job_name = _utils.create_transform_job(client, vars(args))
   logging.info('Batch Job request submitted. Waiting for completion...')
-  _utils.wait_for_transform_job(client, batch_job_name)
+
+  try:
+    _utils.wait_for_transform_job(client, batch_job_name)
+  except:
+    raise
+  finally:
+    cw_client = _utils.get_cloudwatch_client(args.region)
+    _utils.print_logs_for_job(cw_client, '/aws/sagemaker/TransformJobs', batch_job_name)
```

Comment:
can we move this to end of file? after `Batch Transformation creation completed.` This will ensure the outputs have also been written before logs are printed and also be clear to user that logs are not realtime

Reply:
then we would need to keep the whole thing in try block which is unnecessary 
The waiter actually prints saying that the job is complete and then we print logs 
seems like an appropriate place

Reply:
> The waiter actually prints saying that the job is complete 

Sounds fine then. That should be intuitive for user